ext/libyuv.cmd

```diff
@@ -6,6 +6,8 @@
 
 : # If you're running this on Windows, be sure you've already run this (from your VC2019 install dir):
 : #     "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
+: # Clang-cl is recommended to build libyuv on Windows. Run this if you have clang-cl installed:
```

Comment:
Nit: Change the first sentence to "It is recommended to build libyuv with clang-cl on Windows." We can also explain the reason is that MSVC doesn't support the assembly code in libyuv, which is written in the GCC inline assembly syntax.

Reply:
Done. The appveyor build has been pending for 10 hours, but since I only updated comment this should success.